core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/cluster/rds/upgrade/validation/ValidateRdsUpgradeActions.java

```diff
@@ -33,6 +37,7 @@ public class ValidateRdsUpgradeActions {
             @Override
             protected void doExecute(ValidateRdsUpgradeContext context, ValidateRdsUpgradeTriggerRequest payload, Map<Object, Object> variables) {
                 Long stackId = payload.getResourceId();
+                variables.put(TARGET_MAJOR_VERSION_KEY, payload.getVersion());
```

Comment:
nit: It would be a little cleaner having this in the context as it is in the following, but it is ok as it is if it would be too much effort:

```
public class UpgradeRdsContext extends ClusterViewContext {

    private final TargetMajorVersion version;
```

Reply:
As we discussed, it would be left as it is, as the change would be to big (we should pass the targetmajorversion on all the flow requests / responses).